ui/src/components/SecretsTable/index.jsx

```diff
@@ -27,7 +27,13 @@ const sorted = pipe(
  * Display secrets in a table.
  */
 export default class SecretsTable extends Component {
+  static defaultProps = {
```

Comment:
Default props without knowing the actual props can be confusing. Not too important, but if we can have `propTypes` come before `defaultProps`, it would be great :)